cookbooks/bcpc-hadoop/recipes/oozie.rb

```diff
@@ -181,9 +183,9 @@
 ruby_block "Oozie Down" do
   i = 0
   block do
-    status=`sudo -u oozie oozie admin -oozie http://"#{float_host(node[:fqdn])}":11000/oozie -status 2>&1` 
+    status=`sudo -u oozie oozie admin -oozie #{OOZIE_URL} -status 2>&1` 
     while not /NORMAL/ =~ status and $?.to_i
-      status=`sudo -u oozie oozie admin -oozie http://"#{float_host(node[:fqdn])}":11000/oozie -status 2>&1` 
+      status=`sudo -u oozie oozie admin -oozie #{OOZIE_URL} -status 2>&1` 
```

Comment:
Maybe create a function or some sort of definition for testing that Oozie's alive? This is the second instance of this status check code in the recipe.